lib/ome/files/out/MinimalTIFFWriter.cpp

```diff
@@ -233,18 +247,35 @@ namespace ome
       void
       MinimalTIFFWriter::setupIFD() const
       {
-        // Default to single strips for now.
         ifd->setImageWidth(getSizeX());
         ifd->setImageHeight(getSizeY());
 
-        // Default strip size.  We base this upon a default
+        auto tile_x = getTileSizeX();
+        auto tile_y = getTileSizeY();
+
+        // Default strip or tile size.  We base this upon a default
         // chunk size of 64KiB for greyscale images, which will
-        // increase to 192KiB for 3 sample RGB images.
+        // increase to 192KiB for 3 sample RGB images.  We use strips
+        // up to a width of 2048 after which tiles are used.
         if(getSizeX() == 0)
           {
             throw FormatException("Can't set strip or tile size: SizeX is 0");
           }
-        else
+        else if(!tile_x && tile_y)
```

Comment:
- tile_x set && tile_y set == tiles
- tile_x unset && tile_y set == strips
- tile_x set && tile_y unset == invalid, so we fall back to the automatic strip/tile behaviour below